app/views/kuroko2/layouts/application.html.slim

```diff
@@ -59,7 +59,7 @@ html
 
       aside.right-side
         section.content-header
-          h1= raw(@content_title || @title)
+          h1= raw yield(:content_title) || yield(:title)
```

Comment:
`yield(:undefined_symbol)` returns `""` (empty string).

So `yield(:content_title) || yield(:title)` returns always `yield(:content_title)` value.
Do you want `yield(:content_title).presence || yield(:title)` here?